gdcdictionary/schemas/observation.yaml

```diff
@@ -238,9 +240,10 @@ properties:
 
   onset_unk:
     description: >
-      If symptomatic, onset date - unknown
-    type: boolean
-
+      Check "Unknown", if symptomatic but onset date is unknown.
```

Comment:
I think I'm just being picky, but not sure a "," is needed, it reads like Check "Unknown". But only if....

Reply:
Resolved